geometry/geometry/TrianglesView.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  View1.swift
+//  geometry
+//
+//  Created by Dmitry Tolmachev on 30.11.15.
+//  Copyright © 2015 Al Digit. All rights reserved.
+//
+
+import UIKit
+
+class TrianglesView: UIView {
+    private var dataSource: TrianglesProtocol?
+    
+    override func drawRect(rect: CGRect) {
+        UIColor.greenColor().set()
+        let context = UIGraphicsGetCurrentContext()
+        CGContextSetLineWidth(context, 3)
+        
+        if let trianglesDict = dataSource?.getTriangles(){
+            var n = 0
+            var numbers = [1:"1",2:"2",3:"3"]
+            
+            for n in 1...3 {
+            CGContextMoveToPoint(context, (trianglesDict[numbers[n]!]!["first"]!!["x"] as! CGFloat), (trianglesDict[numbers[n]!]!["first"]!!["y"] as! CGFloat))
```

Comment:
Probably part of a old code returne in program. I return right code back,
is like short

2016-06-06 10:04 GMT+06:00 psumarokov <notifications@github.com>:

> In geometry/geometry/TrianglesView.swift
> <https://github.com/todem5/geometry/pull/18#discussion_r65835437>:
>
> > +import UIKit
> > +
> > +class TrianglesView: UIView {
> > +    private var dataSource: TrianglesProtocol?
> > +
> > +    override func drawRect(rect: CGRect) {
> > +        UIColor.greenColor().set()
> > +        let context = UIGraphicsGetCurrentContext()
> > +        CGContextSetLineWidth(context, 3)
> > +
> > +        if let trianglesDict = dataSource?.getTriangles(){
> > +            var n = 0
> > +            var numbers = [1:"1",2:"2",3:"3"]
> > +
> > +            for n in 1...3 {
> > +            CGContextMoveToPoint(context, (trianglesDict[numbers[n]!]!["first"]!!["x"] as! CGFloat), (trianglesDict[numbers[n]!]!["first"]!!["y"] as! CGFloat))
>
> This has been refactored to a more shorter version when using arrays
> instead of dictionaries.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/todem5/geometry/pull/18/files/7c2c5af31f022ce318230665e7dc3f55933a31e6#r65835437>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ALUaDlYGT0LdS4yVkYVp2wae2RNbcObFks5qI5w8gaJpZM4IuX-s>
> .
>



-- 
C Уважением Дмитрий!